package-lock.json

```diff
@@ -86,20 +115,20 @@
       "resolved": "https://registry.npmjs.org/form-data/-/form-data-0.0.8.tgz",
       "integrity": "sha1-CJDNEAXFzOzAudJKiAUskkQtDbU=",
       "requires": {
-        "async": "0.2.10",
-        "combined-stream": "0.0.7",
-        "mime": "1.2.11"
+        "async": "~0.2.7",
```

Comment:
Why did these change? I generally like to pin versions in the package-lock file.

Reply:
Thanks for the tip! I changed my npm configs; `save-exact` to `true` & `save-prefix` to "~". Not sure how to prevent the package-lock changes though. From some quick research the solution seems to be using npm 5.

Either way, I've manually fixed *package-lock.json*.